code/script.js

```diff
@@ -26,22 +32,117 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+
   chat.scrollTop = chat.scrollHeight
 }
 
+//This function makes the conversation happen
+const conversation = (message) => {
+  if (questionNumber === 1) {
+    setTimeout(() => greeting(message, 'bot'), 1000)
+
+  } else if (questionNumber === 2) {
+    showMessage(message, 'user') 
+    handleNameInput.value = ''
+    setTimeout(() => drinks(message, 'user'), 1000)
+
+  } else if (questionNumber === 3) {
+    showMessage(message, 'user')
+    setTimeout(() => drinkChoices(message), 1000)
+
+  } else (questionNumber === 4)
+}
+
+
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  questionNumber = 1
```

Comment:
This is good solution if the number of questions is reduced, like in this case. But we're curious: Did you also consider using `questionNumber++` instead?